src/components/experiment-tracking/experiment-primary-toolbar/index.js

```diff
@@ -1,15 +1,31 @@
 import React from 'react';
 import PrimaryToolbar from '../../primary-toolbar';
+import IconButton from '../../icon-button';
+import ShowChangesIcon from '../../icons/show-changes';
 
 export const ExperimentPrimaryToolbar = ({
   sidebarVisible,
   setSidebarVisible,
+  enableShowChanges,
+  setEnableShowChanges,
+  enableComparisonView,
+  showChangesIconDisabled,
 }) => {
   return (
     <PrimaryToolbar
       visible={{ sidebar: sidebarVisible }}
       onToggleSidebar={setSidebarVisible}
-    />
+    >
+      <IconButton
+        ariaLive="polite"
+        className={'pipeline-menu-button--labels'}
+        onClick={() => setEnableShowChanges(!enableShowChanges)}
+        icon={ShowChangesIcon}
+        labelText={`${enableShowChanges ? 'Disable' : 'Enable'} Show Changes`}
```

Comment:
We generally do sentence case here, so I think this should be:

```suggestion
        labelText={`${enableShowChanges ? 'Disable' : 'Enable'} show changes`}
```

For example:

![image](https://user-images.githubusercontent.com/16869061/145395630-108241fd-8086-4671-8625-0b346d6e34fa.png)